src/libraries/tests.proj

```diff
@@ -169,6 +173,10 @@
                       Exclude="@(ProjectExclusions)"
                       BuildInParallel="false"
                       Condition="'$(TestTrimming)' == 'true'" />
+    <ProjectReference Include="..\mono\netcore\sample\iOS\Program.csproj"
+                      Exclude="@(ProjectExclusions)"
+                      BuildInParallel="false"
+                      Condition="'$(ArchiveTests)' == 'true'" />
```

Comment:
this can be simplified since we don't really need the ProjectExclusions (the reason for those is to filter out individual projects from a **\*.Tests.csproj wildcard, but we're only referencing a single project here so we don't need that):

```suggestion
    <ProjectReference Include="$(MonoProjectRoot)\netcore\sample\iOS\Program.csproj"
                      BuildInParallel="false"
                      Condition="'$(ArchiveTests)' == 'true' and ('$(TargetOS)' == 'iOS' or '$(TargetOS)' == 'tvOS')" />
```

Reply:
Updated, thanks!